docs/user-docs/latest/settings/managing-global-settings.md

```diff
@@ -131,3 +131,7 @@ To create a new API token:
 ::: caution
 For security reasons, API tokens are only shown right after they have been created. When refreshing the page or navigating elsewhere in the admin panel, the newly created API token will be hidden and will not be displayed again.
 :::
+
+## Configuring other plugins
+
+Installed plugins can add their own settings sub-sections. These sections are found at the bottom of the list of sub-sections, below the settings for Strapi built-in plugins. Settings for 3rd party plugins are described in the plugin's documentation on the [Marketplace](https://market.strapi.io).
```

Comment:
minor point: in the User Guide intro to plugins we refer to 'pre-installed' plugins. Here you use 'built-in' are they the same thing? If so should the word choice be harmonized?

Reply:
Indeed! I was mostly trying to find a generic term for "all the Strapi-maintained plugins that are already shipped with the product, either installed or not." But indeed, I think new settings by "other" plugins will always be injected at the end of the list, so they will always appear after existing (i.e., pre-installed) plugins.